### ISSUE TITLE
EOL Google Play Music script

### ISSUE THREAD

**Issue by andrewazores** [tiliado/nuvolaruntime#660]
https://www.androidauthority.com/google-play-music-shutting-down-1170984/

Various other reports are also out as of today that GPM is finally being shut down in a rolling fashion and users told to finish their migration to YouTube Music. The GPM script is therefore obsolete ( :-( ) ...

**Comment by fenryxo** [comment]
Indeed. It's quite said as Google Play Music was the very first and the only script in *Google Music Frame* before additional scripts were added and the app renamed to *Nuvola Player*.
